src/forgot-password/LoadableForgotPasswordPage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import loadable from '@loadable/component';
+import Spinner from '../common-components/Spinner';
+
+const LoadableComponent = loadable(() => import('./ForgotPasswordPage.jsx'), {
+  fallback: <Spinner />,
+});
+
+const LoadableForgotPasswordPage = () => <LoadableComponent />;
+
+export default LoadableForgotPasswordPage;
```

Comment:
do we need separate component for it? Can't we add this code directly below the ForgotPasswordPage.jsx and just export that instead of exporting `ForgotPasswordPage`

Reply:
@zainab-amir I guess not, because in that case, the app will load the actual code as well while importing all these components in `index.jsx` which actually we want to avoid here.

Reply:
It also makes code cleaner to keep them separate.